README.md

```diff
@@ -23,19 +23,68 @@ English | [简体中文](./README.zh-CN.md)
 
 Dashboard for [Apache APISIX](https://github.com/apache/apisix)
 
-## User Guide
+The goal of Apache APISIX Dashboard project is to enable everyone to quickly experience and learn Apache APISIX, and it still cannot be used directly in a production environment. its function point coverage is always a subset of Apache APISIX, and may lag behind the rapid iteration of Apache APISIX. 
 
-Please refer to [User Guide](./docs/USER_GUIDE.md)
+If you need to use the Dashboard project in a production system, you need to enhance user permissions, communication security, high availability and other advanced features.
+
+## Install
+
+Support multiple ways to install APISIX dashboard
+
+### Docker
+
+Start a runnable version by the following method
+
+- [Deploy With Docker](./docs/deploy-with-docker.md)
+
+### Build From Source Code
+
+To build from source code, first make sure that your `golang` version is 1.13 or greater.
+Also you need to follow the `node` and `yarn` in advance
+
+```
+$ git clone -b v2.0 https://github.com/apache/apisix-dashboard.git && cd apisix-dashboard
+$ make build
+```
+
+Then you can find all files (configuration files, executable files, web static resources) needed to run dashboard in the `./output` directory.
+
+Start by the following command
 
-## Deployment
+```sh
+$ cd ./output
+$ export ENV=local && exec ./manager-api
+```
 
-- [Deploy Manually](./docs/deploy.md)
-- [Deploy with Docker](./docs/deploy-with-docker.md)
+`makefile` provides the following commands
 
-## Development
+```
+Makefile rules:
 
-- [Apache APISIX](https://github.com/apache/apisix)
-- [Dashboard](./docs/develop.md)
+    help:		    Show Makefile rules
+    build:		    build dashboard, it contains web and manager-api
+    api-test:		Run the tests of manager-api
+    api-run:		Run the manager-api
+    api-stop:		stop manager-api
+    go-lint:    	Lint Go source code
+    license-check:	Check apisix-dashboard source codes for Apache License
+```
+
+For more detailed construction steps, see -  [Build from source code](./docs/deploy.md)
```

Comment:
Why not go to here?

https://github.com/apache/apisix-dashboard/pull/729/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R40

Reply:
`deploy.md` has more detail config， keep simple in readme